### DISCUSSION TITLE
How can I use eslint for astro files?

### DISCUSSION THREAD

**Discussion by tinchoz49** [zed-industries/zed#9885] [Feature Requests]
Hey everyone, so, we have astro projects that are being validated using eslint. Is it possible to setup eslint for astro files instead of using prettier?

Thanks!

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
